lib/js/jsbigints.nim

```diff
@@ -0,0 +1,132 @@
+## Arbitrary precision integers.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+type JsBigInt* = ref object of JsRoot ## Arbitrary precision integer for JavaScript target.
```

Comment:
Error: undeclared identifier: 'JsRoot'

Reply:
https://nim-lang.github.io/Nim/system.html#JsRoot

Reply:
I know, what I mean is to add:
`static: doAssert defined(js)`

so that you get a good error message when trying to use this in non js backend